integration-tests/tests/preview-dataset.js

```diff
@@ -16,13 +16,10 @@ const testPreviewDatasetFn = withUserToken(async ({ testUrl, page, token }) => {
   await click(page, clickable({ textContains: 'Preview data' }))
   await waitForNoSpinners(page)
 
-  const previewTableName = 'Participant Preview Data'
-  await findTableCellText(page, getTableHeaderPath(previewTableName, 1), 'participant_id')
-  await findTableCellText(page, getTableHeaderPath(previewTableName, 2), 'biological_sex')
-  await findTableCellText(page, getTableHeaderPath(previewTableName, 3), 'age')
-  await findTableCellText(page, getTableCellPath(previewTableName, 2, 1), 'participant1')
-  await findTableCellText(page, getTableCellPath(previewTableName, 2, 2), 'male')
-  await findTableCellText(page, getTableCellPath(previewTableName, 2, 3), '36')
+  const tableName = 'Participant Preview Data'
+  await findTableTextWithinColumn(page, { tableName, columnHeader: 'participant_id', textContains: 'participant1' })
```

Comment:
I think it's useful to search by column name, but should check to ensure that a particular row of the table has the right data? For example, perhaps we can select the row by a column value (`participant_id` is `2`), and then ensure that row 2 indeed contains `2`, `female`, `33`:

```
participant_id | biological_sex | age
---------------+----------------+----
1              | male           | 26
2              | female         | 33
```